forge-gui/res/cardsfolder/l/loose_in_the_park.txt

```diff
@@ -0,0 +1,13 @@
+Name:Loose in the Park
+ManaCost:1 G
+Types:Enchantment Aura
+K:Enchant land
+A:SP$ Attach | ValidTgts$ Land | AILogic$ Pump
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ FreeCard | TriggerDescription$ When CARDNAME enters the battlefield, draw a card.
```

Comment:
I think if you look at this card in a game (you can use Dev Mode to speed things up) you will see that it's missing a part of the text
Unlike SpellDescription in most cases TriggerDescription should be completely done on the main trigger line